src/setup/rgw.py

```diff
@@ -178,6 +178,20 @@ def init(conf: MappedConf):
             Conf.delete(rgw_consul_idx, rgw_lock_key)
             Log.info(f'{rgw_lock_key} key is deleted')
 
+        Log.info('Starting radosgw service.')
```

Comment:
Could you please create a separate class for entrypoint logic ? 
in future if we want to add more logic to entrypoint then we should be easily isolate entry point code.

Reply:
Agree, separate function for user creation is also needed

Reply:
Separate function for user creation already exists. Separate class for entry point is added